resources/services/elasticbeanstalk/environments.go

```diff
@@ -433,6 +435,12 @@ func resolveElasticbeanstalkEnvironmentListeners(ctx context.Context, meta schem
 		ResourceArn: p.EnvironmentArn,
 	}, func(o *elasticbeanstalk.Options) {})
 	if err != nil {
+		// It takes a few minutes for an environment to be terminated
```

Comment:
Fix for https://github.com/cloudquery/cq-provider-aws/issues/621#issuecomment-1103846841

Reply:
Please use `c.IsNotFoundError(err)` instead

Reply:
Thanks for the tip! Done in https://github.com/cloudquery/cq-provider-aws/pull/751/commits/28ffc78101822538c840b3f7cc59a7fffc6346f3